votor/src/certificate_pool.rs

```diff
@@ -555,11 +556,13 @@ impl CertificatePool {
     /// the size of the certificate
     #[cfg(test)]
     pub fn slot_notarized_fallback(&self, slot: Slot) -> Option<usize> {
+        // TODO: remove this line once `CertificateMessage` is moved into the repo`
+        use crate::certificate_pool::vote_certificate::get_vote_count;
```

Comment:
I don't think we need get_vote_count any more, it was only used in tests and those tests have since been removed, I'm removing it in https://github.com/anza-xyz/alpenglow/pull/300